### DISCUSSION TITLE
Vercel storage giving me 500: Internal Server Error

### DISCUSSION THREAD

**Discussion by dinakart** [vercel/community#2529] [Bug Report]
### Summary

I am trying to access my postgres psql command and receiving a full white screen when I click on my db under the storage tab

500: Internal Server Error
Code: INTERNAL_REGIONAL_CACHE_FAILED
ID: cle1::tnc4v-1684595874954-2dead55e37f9
<img width="1273" alt="Screenshot 2023-05-20 at 11 18 17 AM" src="https://github.com/vercel/community/assets/45085331/4483cf9c-5545-49ad-b67c-9b027cc37392">

Does anyone know why this could be?

**Comment by ttsoares** [comment]
YES!  Same here... I'm fighting the issue in the last 4 days !!!
My APP works locally (in my computer) talking normally with the DB (Postgres) in Vercel... 
**But the SAME application after deployment is unable to talk with the same DB IN Vercel !**
The error message gives no indication about the source of the problem.
Today I tried two different 'fetch' implementations but with no avail.
It seems that will need to deploy in some other place but pointing the DB in Vercel  :-(
Also - to get nuts - I was able to make it work ONCE but after that the next deploy break again...

**Comment by amyegan** [comment]
Hi @dinakart and @ttsoares. Can you share the exact steps I should take to reproduce this error?

**Reply by ttsoares** [reply]
I commented here because this issue is similar to the one I reported here:
[https://github.com/orgs/vercel/discussions/2490](https://github.com/orgs/vercel/discussions/2490)
But - for my case - I found a solution which is the last post closing that issue...
Maybe that approach could help this particular case here also.
